src/components/FindPark.vue

```diff
@@ -13,12 +13,15 @@
         Find a dog park near me!
       </button>
     </router-link>
-    <h3>--Or--</h3>
+    <h3 id="location-message" v-if="!this.$store.state.geolocation">
+      Turn on location services and reload the page to search for parks near you!
+    </h3>
+    <h3 id="search-switch">--Or--</h3>
     <input
       class="search-input"
       type="text"
       aria-label="Type your search terms here"
-      placeholder="Enter name, city or zip code to search"
+      placeholder="Enter city and state to search"
```

Comment:
Thanks for this clarification for the user so that they don't try searching by address or zip code!